### ISSUE TITLE
Multiple languages missing from language picker

### ISSUE THREAD

**Issue by salarua** [elk-zone/elk#1760]
Several languages supported by Mastodon-FE are missing from Elk.

The specific languages missing are:
- Asturian
- Sorani
- Montenegrin
- Lojban
- Kabyle
- Kurmanji
- Láadan
- Lingua Franca Nova
- Scots
- Southern Sami
- Lule Sami
- Silesian
- Tai
- Toki Pona
- Balaibalan
- Standard Moroccan Tamazight

**Comment by salarua** [comment]
I finally looked and I saw that the languages for the post language selector are simply imported from somewhere else. This may be an upstream issue.

**Comment by shuuji3** [comment]
Yes, the languages in the post language selector came from npm library named `iso-639-1` (ref. https://github.com/shuuji3/elk/blob/8c881f4d904b9e9eddd8359005d1388820fc9701/composables/langugage.ts).

Maybe because some languages are not included 639-1: https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes

**Comment by shuuji3** [comment]
Mastodon is hardcoding some of the languages from ISO-639-3: https://github.com/mastodon/mastodon/blob/e675c10d1926fcc8801813cfdb5be5bd58392c2d/app/helpers/languages_helper.rb#L191-L207.

**Comment by zordsdavini** [comment]
Also from ISO 639-3:
- Samogitian (sgs) - žemaitiu
- Latgalian (ltg) - latgalīšu
- Prussian (prg) - prūsiskai
